### PR TITLE
uic fall 2019 update

### PR DESCRIPTION
**Pull request by kaytwo**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

Up to date adjunct list for non-CS primary department folks: https://cs.uic.edu/faculty-staff/adjunct-faculty/. If the primary faculty list is needed for verification, it is available at https://cs.uic.edu/faculty-staff/faculty/.

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- (N/A) If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- (N/A) If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -6703,6 +6703,7 @@ James J. Martin,Clemson University,https://people.cs.clemson.edu/~jmarty,66VE-3M
 James K. Baker,Carnegie Mellon University,http://ethw.org/James_K._Baker,9Z2rJjYAAAAJ
 James K. Hahn,George Washington University,https://www.seas.gwu.edu/~hahn,m0BU16YAAAAJ
 James Kwangjune Hahn,George Washington University,https://www.seas.gwu.edu/~hahn,m0BU16YAAAAJ
+James L. Patton,University of Illinois at Chicago,https://pattonj.people.uic.edu/,GWUT6MQAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: Not in CS and no mention of an affiliation.

**Review by emeryberger — CHANGES_REQUESTED** [review]
Please remove non-eligible faculty.

File: csrankings.csv (hunk 1 above)
Comment (kaytwo) [reply csrankings.csv]: Prof. Patton is listed on https://cs.uic.edu/faculty-staff/adjunct-faculty/ as adjunct faculty (can solely advise CS PhD students). Is it also required that he list this affiliation on his own page?

Comment (emeryberger) [reply csrankings.csv]: Nope, that's fine - thanks!

Comment (kaytwo) [reply csrankings.csv]: These diffs went stale so I rebased them - should be clean to merge now.

**Comment by kaytwo** [comment]
Hi again @emeryberger , I've made the requested changes and believe this PR is ready to be merged.

**Review by emeryberger — COMMENTED** [review]
LGTM

**Pull request merged**